PayPalNativeCheckout/src/main/java/com/braintreepayments/api/PayPalNativeCheckoutRequest.java

```diff
@@ -0,0 +1,17 @@
+package com.braintreepayments.api;
+
+import androidx.annotation.NonNull;
+
+public class PayPalNativeCheckoutRequest {
```

Comment:
Do we not need a return URL for Android? 👀

Reply:
Good callout. We _do_ but it's technically in the `AndroidManifest.xml`. We may end up adding one down the line, but for now we can get it from `BraintreeClient`.

Reply:
Ah, gotcha gotcha, that makes sense!